docs/quick-install.md

```diff
@@ -3,7 +3,7 @@
 
 SignalFx Smart Agent Integration installs the Smart Agent application on a single host machine from which you want to collect monitoring data. Smart Agent collects infrastructure monitoring, µAPM, and Kubernetes data.
 
-For other installation options, including bulk deployments, see [Advanced Installation Options](./advanced-install-options.md).
+For other installation options, including bulk deployments, see [Install and configure the Smart Agent](../agent-install-methods.html).
```

Comment:
@jmalin-signalfx `../agent-install-methods.html` does not currently exist in this repo.  Also, I believe that this link will be auto converted to `https://docs.signalfx.com/en/latest/integrations/agent/../agent-install-methods.html` when synced to the integrations/product-docs repos, so we need to make sure that this link will be valid.

Reply:
It doesn't exist in the signalfx-agent repo.
It resolves to https://docs.signalfx.com/en/latest/integrations/agent/agent-install-methods.html when synced. I have confirmed this by looking at the Read The Docs preview site.

I don't know how your auto-conversion works. You'll have to advise me.

Reply:
The sync script currently prepends `https://docs.signalfx.com/en/latest/integrations/agent/` to relative links, so this will end up getting converted to `https://docs.signalfx.com/en/latest/integrations/agent/../agent-install-methods.html`, which resolves to https://docs.signalfx.com/en/latest/integrations/agent-install-methods.html.

To be safe, I suggest changing the relative links that do not exist in this repo to absolute links so that we can avoid any conversion issues.

Reply:
I'll do this, but it may break the links in our _preview_ builds.